radixdlt-java/src/main/java/com/radixdlt/client/core/network/RadixJsonRpcClient.java

```diff
@@ -99,7 +95,7 @@ public String getLocation() {
 		return wsClient.getEndpoint().url().toString();
 	}
 
-	public Observable<RadixClientStatus> getStatus() {
+	public BehaviorSubject<RadixClientStatus> getStatus() {
```

Comment:
We shouldn't be exposing Subjects to callers of this method as they will then be able to call `onNext`, `onSuccess` etc. which totally breaks contract.

Reply:
Ha, yeah, I noticed that as well while doing the tests. "Why can I call onNext? Ohhhhhh.." :D